examples/prom/de_parametric_maxwell_greedy.cpp

```diff
@@ -0,0 +1,956 @@
+                   "--no-useAmgX",
+                   "Enable or disable AmgX in MatrixFreeAMS.");
+#endif
+    args.AddOption(&fom, "-fom", "--fom", "-no-fom", "--no-fom",
+                   "Enable or disable the fom phase.");
+    args.AddOption(&offline, "-offline", "--offline", "-no-offline","--no-offline",
+                   "Enable or disable the offline phase.");
+    args.AddOption(&online, "-online", "--online", "-no-online", "--no-online",
+                   "Enable or disable the online phase.");
+    args.AddOption(&build_database, "-build_database", "--build_database",
+                   "-no-build_database", "--no-build_database",
+                   "Enable or disable the build_database phase of the greedy algorithm.");
+    args.AddOption(&use_database, "-use_database", "--use_database",
+                   "-no-use_database", "--no-use_database",
+                   "Enable or disable the use_database phase of the greedy algorithm.");
+    args.AddOption(&greedy_param_space_min, "-greedy-param-min",
+                   "--greedy-param-min", "The minimum value of the parameter point space.");
+    args.AddOption(&greedy_param_space_max, "-greedy-param-max",
+                   "--greedy-param-max", "The maximum value of the parameter point space.");
+    args.AddOption(&greedy_param_space_size, "-greedy-param-size",
+                   "--greedy-param-size",
+                   "The number of values to search in the parameter point space.");
+    args.AddOption(&greedy_relative_error_tol, "-greedyrelerrortol",
+                   "--greedyrelerrortol", "The greedy algorithm relative error tolerance.");
+    args.AddOption(&greedy_subset_size, "-greedysubsize", "--greedysubsize",
+                   "The greedy algorithm subset size.");
+    args.AddOption(&greedy_convergence_subset_size, "-greedyconvsize",
+                   "--greedyconvsize", "The greedy algorithm convergence subset size.");
+    args.AddOption(&de_min_freq, "-de_min_freq", "--de_min_frequency",
+                   "DE min freq.");
+    args.AddOption(&de_max_freq, "-de_max_freq", "--de_max_frequency",
+                   "DE max freq.");
+    args.AddOption(&de_F, "-de_f", "--de_f",
+                   "DE F.");
+    args.AddOption(&de_CR, "-de_cr", "--de_cr",
+                   "DE CR.");
+    args.AddOption(&de_PS, "-de_ps", "--de_ps",
+                   "DE Population size.");
+    args.AddOption(&de_min_iter, "-de_min_iter", "--de_min_iter",
+                   "DE Minimum number of iterations.");
+    args.AddOption(&de_max_iter, "-de_max_iter", "--de_max_iter",
+                   "DE Maximum number of iterations.");
+    args.AddOption(&de_ct, "-de_ct", "--de_ct",
+                   "DE Cost threshold.");
+    args.AddOption(&de, "-de", "--de", "-no-de", "--no-de",
+                   "Enable or disable the differential evolution phase.");
+    args.AddOption(&baseoutputname, "-out", "--outputfile-name",
```

Comment:
I didn't see `baseoutputname` being used anywhere - is the implementation of this option missing or should it get removed? I think it would be useful to have this option to control the file output.

Reply:
This was a leftover copy/paste from the existing DE example `de_parametric_heat_conduction_greedy`, and the option is not enabled in this file (or the other example). I think I am just going to remove the option